web/common/fetchers.ts

```diff
@@ -0,0 +1,41 @@
+import { API_URL } from './constants';
+ 
+export const saveSession = async (dehydratedState: string) => {
+  await fetch(API_URL + '/api/session/save', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ dehydratedState }),
+  });
+};
+ 
+export const destroySession = async () => {
+  try {
+    await fetch(API_URL + '/api/session/destroy', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: null,
+    });
+  } catch (e) {
+    console.log('ERROR:', e);
+  }
+};
+
+export const createUser = async (formData: Object) => {
+  await fetch(API_URL + '/api/user/create', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(formData),
+  });
+};
+
+export const createDao = async (formData: Object) => {
```

Comment:
PP: change filename as this method will post data :P

Reply:
well, the `fetchers` refer to the `fetch` library which does both POST and GET calls, but I can rename it if you want :D